frontend/src/components/core/MainMenu/MainMenu.tsx

```diff
@@ -426,27 +587,20 @@ function MainMenu(props: Props): ReactElement {
       }}
       placement={PLACEMENT.bottomRight}
       content={({ close }) => (
-        <StatefulMenu
-          items={menuOptions}
-          onItemSelect={({ item }) => {
-            item.onClick()
-            close()
-          }}
-          overrides={{
-            Option: MenuListItem,
-            List: {
-              props: {
-                "data-testid": "main-menu-list",
-              },
-              style: {
-                ":focus": {
-                  outline: "none",
-                },
-                border: `1px solid ${colors.fadedText10}`,
-              },
-            },
-          }}
-        />
+        <>
```

Comment:
What is with this nameless element? And do we need it?

Reply:
I tried removing the nameless element and it looks like the logic for the second submenu showing up (
 will not work without the fragment.

Reply:
The fragment here is needed because the return value for the function isn't valid `tsx` without it (as it ends up being two tsx elements without a parent)